app/src/main/java/ru/suslovalex/androidacademyapp/adapters/AdapterActors.kt

```diff
@@ -0,0 +1,43 @@
+package ru.suslovalex.androidacademyapp.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import ru.suslovalex.androidacademyapp.R
+import ru.suslovalex.androidacademyapp.model.Actor
+
+class AdapterActors: RecyclerView.Adapter<ActorViewHolder>() {
+
+    private var actors: List<Actor> = mutableListOf()
```

Comment:
@AleksandrAlex Please move to constarctor

Reply:
Same reason. I wrote it above in app/src/main/java/ru/suslovalex/androidacademyapp/FragmentMoviesList.kt
Please will write about my opinion)